model-optimizer/extensions/back/ReverseInputChannels.py

```diff
@@ -146,7 +146,7 @@ def pass_rc_through_transpose(node: Node, reverse_channels: Node):
         if order is None:
             return False
         new_axis = list(order).index(reverse_axis)
-        reverse_channels.axis = new_axis
+        reverse_channels.axis = int64_array(new_axis)
```

Comment:
May we obtain a problem with code lines like `if shape[reverse_channels.axis] == 1:`? Is reverse_channels.axis always 0D array?

Reply:
In MO we set only 0D arrays as axis. There is also a transformation [UnpackPackReverseInputChannels](https://github.com/openvinotoolkit/openvino/blob/master/model-optimizer/extensions/front/tf/UnpackPackReverseInputChannels.py#L67) which inserts ReverseChannels with axis from [Stack](https://www.tensorflow.org/api_docs/python/tf/stack) and [Unstack](https://www.tensorflow.org/api_docs/python/tf/unstack) operations. Axis is int in both operartions.

But still if axis will be not a 0D tensor then pass_rc_through_transpose(), lift_up_through_transpose() will fail with errors. So I think check is needed.

Reply:
Added checks in pass_rc_through_transpose(), lift_up_through_transpose().

Reply:
Discussed offline, removed checks as they are not needed.